src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -472,10 +472,8 @@ public function getName()
     {
         if (empty($this->nname)) {
             return $this->fname.' '.$this->sname;
-        } else {
-            return $this->fname.' "'.$this->nname.'" '.$this->sname;
-        }
-        return $this->fname.' '.$this->nname.' '.$this->sname;
+        } 
+        return $this->fname.' "'.$this->nname.'" '.$this->sname;
```

Comment:
it feels like this case should be inverted, as having a nickname set is definitely the non-default case